contributors/guide/contributor-cheatsheet/README-ko.md

```diff
@@ -205,7 +205,7 @@ GitHub 이슈는 버그 리포트, 개선 요청 또는 실패한 테스트와 
 - 더 자세한 설명이 필요하다고 생각하는 자신의 PR에 대한
   의견을 말한다.
 - [`/assign @<username>`][assign]으로 PR을 선택적으로 할당한다.
-  많은 리뷰어를 지정하더라도, 더 빨리 PR을 검토되지는 않는다.
+  많은 리뷰어를 지정하더라도, 더 빨리 PR이 검토되지는 않는다.
```

Comment:
```suggestion
  리뷰어를 많이 지정하더라도, PR이 더 빨리 검토되지는 않는다.
```

Reply:
Resolved in a74a925c74cd58f117ef6fe9821d3d5724fa8063